scans/worker.py

```diff
@@ -0,0 +1,41 @@
+import os
+import json
+import boto3
+dynamodb = boto3.resource('dynamodb')
```

Comment:
Consider making region an environment variable in `serverless.yml`, and access it from here in runtime. Some AWS resources require `region` parameter, plus it could make your code more portable.